READMEs/marketplace-flow.md

```diff
@@ -192,14 +193,14 @@ erc20_token = ocean.get_datatoken(asset.get_service("access").datatoken)
 OCEAN_token = ocean.get_datatoken(ocean.OCEAN_address)
 
 ss_params = [
-    ocean.web3.toWei(1, "ether"),
```

Comment:
In this particular case, we want a to_wei function inside Ocean class, to avoid imports. The main advantage of the toWei was that we could call it directly from ocean. When we discussed this refactor, we also discussed placing a to_wei utility inside the Ocean class and using that where convenient. Please add that as discussed.